pandunia-loge.csv

```diff
@@ -2471,98 +2471,96 @@ meza tela||||tablecloth|nappe|mantel|toalha de mesa|скатерть|غِطَا
+be mirde||||fade|||||||||||||||||niknąć (blaknąć, gasnąć, słabnąć)
+mira plate|||eng:mirror, fra:miroir, ara:mirʾāh, arz:mirāya|mirror||espejo (retrovisor)|||||||||||||spegulo|peili|lustro
+mira sonde||||echo||eco|||||||||||||eĥo|kaiku|echo
 misal|||hin:मिसाल (misāl), ara:(mithāl), fas:(mesâl), may:misal, swa:mathalani, tur:mesela|example (instance of)||ejemplo|||||||||||mfrano||ekzemplo|esimerkki|przykład
-misalo||||for example|par exemple|por ejemplo|por exemplo|например||比如|例えば|예컨대|ví dụ|उदाहरण के लिए||cotohnya (misalnya)|kwa mfano|örneğin (mesela)|ekzemple|esimerkiksi|na przykład
+do misal||||for example|par exemple|por ejemplo|por exemplo|например||比如|例えば|예컨대|ví dụ|उदाहरण के लिए||cotohnya (misalnya)|kwa mfano|örneğin (mesela)|ekzemple|esimerkiksi|na przykład
 mise|||eng:emit, spa:remitir, por:emitir, fra:émettre|send (emit, transmit)||enviar (transmitar)||||||||||||||lähettää|wysłać (wyemitować)
 mise ja||||sender (transmitter)||remitente||||||||||||||lähettäjä (lähetin)|nadawca
 mise she||||mission (transmission, emission)||emisión||||||||||||||lähetys|misja
 mishe|||deu:mischen, rus:мешать (měšat'), pol:mieszać, eng:mix, ita:miscelare, + hin:मिश्रण (miśrān)|mix (blend)||mezclar||мешать|||||||||||miksi|sekoittaa|miksować, mieszać
-misher||||mixer (blender)||batidora (licuadora, mezcladora)|||||||||||||miksilo|sekoitin|mikser, blender, robot kuchenny
-mishi alkol||||cocktail||cóctel||коктейль|||カクテル (混合酒)||||||||||koktajl
-miskini|||ara:مِسْكِين (miskīn), fas:مسکین‎ (meskin), tur:may:miskin, swa:maskini, hin:, ben:মিসকিন (miśkin), fra:miskine, spa:mezquino, por:mesquinho|poor (miserable)||pobre (miserable)|||||||||||||mizera|kurja (raukka)|biedny, mizerny
-misri||||Egyptian||egipcio|||||||||||||egipta|egyptiläinen|egipski
-Misria|desha|EG|ara:(miṣr), tur:Mısır, swa:Misri, hin:मिस्र (misra), ben:মিশর (miśôr)|Egypt|Égypte|Egipto|Egipto (Egito)|Египет||埃及|エジプト|이집트|Ai Cập|मिस्र|মিশর|Mesir|Misri|Mısır|Egipto|Egypti|Egipt
-mita||||meet||encontrar|encontrar|||聚会|会う|||मिलना||||||tavata|spotkać
-mite|||eng:meet, fra:meeting, spa:mitin, jpn:ミーティング (mītingu), hin:मीटिंग (mīṭiṅg)|meeting (gathering)||reunión (junta, mitin)||||会议|会合 (会議, ミーティング)|회합 (회의)||बैठक (सभा, मीटिंग)||||||tapaaminen (kokous)|spotkanie, zebranie
+mishe gi||||mixer (blender)||batidora (licuadora, mezcladora)|||||||||||||miksilo|sekoitin|mikser, blender, robot kuchenny
+mishe alkol||||cocktail||cóctel||коктейль|||カクテル (混合酒)||||||||||koktajl
+miskin|||ara:مِسْكِين (miskīn), fas:مسکین‎ (meskin), tur:may:miskin, swa:maskini, hin:, ben:মিসকিন (miśkin), fra:miskine, spa:mezquino, por:mesquinho|poor (miserable)||pobre (miserable)|||||||||||||mizera|kurja (raukka)|biedny, mizerny
+misri basha|basha|||Egyptian arabic||idioma árabe egipcio|||||||||||||egipta|egyptiläinen|egipski
```

Comment:
**misri di**. Otherwise the translations to Esperanto, Finnish and Polish are misleading.